src/vendor/wordpress/demo/image.twig

```diff
@@ -3,12 +3,13 @@
   style="max-width: 40rem; margin: 0 auto; padding: 0 1.25rem"
 >
   <p>Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Nullam quis risus eget urna mollis ornare vel eu leo. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec id elit non mi porta gravida at eget metus. Vestibulum id ligula porta felis euismod semper.</p>
-  {%
-    if class == 'alignleft' or class == 'aligncenter' or class == 'alignright'
-  %}
-    <div class="wp-block-image">
+  {% if alignment in ['alignleft', 'aligncenter', 'alignright'] %}
+    <div class="wp-block-image {{style}}">
```

Comment:
This is explained in the CSS, but this awkward thing where we're adding a wrapper element for certain alignments is mirroring the behavior of WordPress. I simplified the `if` check and added the block style, since it's always set on the element that gets `wp-block-image`.

Reply:
Nice! I like the `if blah in array` pattern. 👍🏽